tools/packaging/guest-image/build_image.sh

```diff
@@ -99,6 +101,14 @@ main() {
 				initrd_os_version=$(get_from_kata_deps "assets.initrd.architecture.${arch_target}.version")
 				initrd_name="kata-${initrd_distro}-${initrd_os_version}.${image_type}"
 				;;
+			image_initrd_suffix=*)
+				image_initrd_suffix=${OPTARG#*=}
+				if [ "${image_initrd_suffix}" == "sev" ]; then
+					initrd_distro=$(get_from_kata_deps "assets.initrd.architecture.${arch_target}.sev.name")
+					initrd_os_version=$(get_from_kata_deps "assets.initrd.architecture.${arch_target}.sev.version")
+					initrd_name="kata-${initrd_distro}-${initrd_os_version}-${image_initrd_suffix}.${image_type}"
+				fi
```

Comment:
I think the whole logic here could've been replaced by: 6f16071359636d1d8f58d5683d1d269dc43e0521

This would fix the issue that @wainersm mentioned in his last review.

Reply:
Thanks @wainersm and @fidencio.
The latest push should fix it.

Reply:
In my local repo now i see:
`jenkins@sut1:~/VSCodeWorkingDir/kata-containers$  ls -l build/rootfs-initrd-sev/destdir/opt/kata/share/kata-containers/
total 29320
lrwxrwxrwx 1 root root       28 May 10 16:58 kata-containers-initrd-sev.img -> kata-ubuntu-20.04-sev.initrd
-rw-r--r-- 1 root root 30022345 May 10 16:58 kata-ubuntu-20.04-sev.initrd`